src/webapp/features/rabble/GameScreen.tsx

```diff
@@ -34,7 +34,9 @@ const GameScreen = (props: GameBoardProps) => {
   return (
     <div className={styles.board}>
       <div className={styles.topButtonContainer}>
-        <InviteLink gameID={gameID} />
+        {typeof scoreList === "object" && !scoreList[1]?.nickname?.length ? (
```

Comment:
Ah cool, i like the brevity in the second term in this conditional 👍 

interesting to check `object` here, i guess any object would be able to be derefrenced with square brackets!

Reply:
This makes me wonder whether it's best practice to put these old javascripty null checks in typescript code... Let's keep it like it is for now.  Good job cleaning it up 💯